tasks.py

```diff
@@ -423,40 +140,7 @@ def postgresql(c, verbose=False):
         invoke postgresql or invoke postgresql --verbose
 
     """
-    run("echo '*** INSTALLING Postgresql ***'")
-    if PLATFORM == "Linux":
-        if run("command -v psql", warn=True, hide=not verbose):
-            run("echo '===> postgresql {}'".format(MESSAGE_ALREADY_INSTALLED))
-
-        else:
-            if run(
-                "apt-get update && apt-get install postgresql",
-                warn=True,
-                hide=not verbose,
-            ).ok:
-                run('echo "===> postgresql {}"'.format(MESSAGE_OK))
-            else:
-                run('echo "===> postgres {}"'.format(MESSAGE_FAILED))
-        if run("service postgresql start", warn=True, hide=not verbose).ok:
-            run('echo "=====> postgresql successfully started!"')
-        else:
-            run('echo "=====> postgresql failed to start!"')
-        res = run("createdb lookit", warn=True, hide=not verbose)
-        if res.exited == 0:
-            run('echo "Database Successfully created!"')
-        else:
-            run('echo "Database already existed"')
-        if run("python manage.py migrate", warn=True, hide=not verbose).ok:
-            run('echo "====> Migrated Django models to lookit db"')
-        else:
-            run('echo "====> Migration failed"')
-    elif PLATFORM == "Darwin":
-        if run("python manage.py migrate", warn=True, hide=not verbose).ok:
-            run('echo "====> Migrated Django models to lookit db"')
-        else:
-            run('echo "====> Migration failed"')
-    else:
-        run("echo {}".format(MESSAGE_WRONG_PLATFORM))
+    c.run("python manage.py migrate", hide="stdout")
```

Comment:
@kimberscott This is where the migration happens.

Reply:
yep - can we just have the docstring say it's doing the migration (not installing postgres/creating the database anymore)?

Reply:
Whoops, missed that.  Good catch!